ex1/functions.py

```diff
@@ -90,26 +264,149 @@ def f():
 # Define a function named args_to_dict that takes three arguments and returns a
 # dictionary with the position of the argument as the key (starting at 0) and
 # the argument as the value.
+def args_to_dict(arg1, arg2, arg3):
+	"""Makes a dictionary.
+	
+	Takes three arguments and returns a dictionary.
+	
+	Parameters
+	----------
+	arg1, arg2, arg3 : object
+		Arguments that come into the dictionary.
+	
+	Returns
+	-------
+	dict
+		
+	"""
+	return {0: arg1, 1: arg2, 2: arg3}
 
 # BONUS: Write a function named args_to_dict_general that does the same for any
 # number of arguments
-
-# Define a function named lists_to_dict that takes two lists of equal lenght
+def args_to_dict_general(*args):
+	"""Makes a dictionary out of arguments.
+	
+	Makes a dictionary out of any number of given arguments.
+	
+	Parameters
+	----------
+	args : object
+		Various arguments
+		
+	Returns
+	-------
+	d : dict
+		Dictionary consisting of args.
+	"""
+	counter = 0
+	d = {}
+	for x in args:
```

Comment:
This can be solved using `enumerate` instead of the manual increment of the `i` variable.